src/ApiTestCase.php

```diff
@@ -121,6 +127,10 @@ protected static function getKernelClass()
 
     protected function purgeDatabase()
     {
+        if (!$this->isDoctrineSupported) {
+            return;
```

Comment:
I think it would be better to throw some exception here, to inform user, that he wants to purge database without doctrine support turned on.